### ISSUE TITLE
plot depth contours wrong for depth under 1km

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#352]
This is illustrated in the image below.  For example, the "9km" (near 78W 25.5N) should be 0.9km, or 800m (I think the former makes more sense, because using a single unit make more sense to me.

<img width="711" alt="Screen Shot 2021-01-13 at 2 58 16 PM" src="https://user-images.githubusercontent.com/99469/104496833-dff5d580-55af-11eb-94cd-e4a02c5ba524.png">

**Comment by dankelley** [comment]
The error was that I was saying `contour(..., breaks=)` instead of `contour(...,levels=)`: a classic image/contour mixup.

This is fixed in "develop" commit c9eb3d17d9412942fd4e50430202c218e50b1f0c , as exemplified below.

PS. I did not change e.g. "500m" to "0.5km", because the former was the documented behaviour.


``` r
library(argoFloats)
data(index)
topoFile <- oce::download.topo(-82, -71, 23, 30, 2)
topo <- oce::read.topo(topoFile)
par(mar=c(3,3,1,1))
plot(index, bathymetry=list(source=topo, contour=TRUE))
```

![](https://i.imgur.com/yVwPx8T.png)

<sup>Created on 2021-01-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by dankelley** [comment]
Sorry, forgot to close this, even though it had been addressed.
